flow/platforms/asap7/openRoad/pdn/BLOCKS_grid_strategy.tcl

```diff
@@ -30,6 +30,6 @@ add_pdn_connect -grid {top} -layers {M5 M6}
 # Element grid
 ####################################
 # The halo around the macro prevents pdn from blocking pin access
-define_pdn_grid -macro -cells Element -halo "0.25 0.25 0.25 0.25" -voltage_domains {CORE} -name ElementGrid
+define_pdn_grid -macro -cells "Element aes_rcon aes_sbox" -halo "0.25 0.25 0.25 0.25" -voltage_domains {CORE} -name ElementGrid
```

Comment:
Is it possible to use BLOCKS rather than listing the names from different designs here?